content-services/latest/config/repository.md

```diff
@@ -1248,12 +1248,16 @@ cors.preflight.maxage=10
 | scripts.execution.maxStackDepth | The maximum stack depth (call frames) allowed in a single invocation of the interpreter. <br><br>This configuration only works for scripts compiled with interpretive mode, which means the optimization level will always be `-1`, overriding the value from the `scripts.execution.optimizationLevel` property. <br><br>As the interpreter doesn't use the Java stack but rather manages its own stack in the heap memory, a **runaway recursion** in interpreted code would eventually consume all available memory and cause an error. This setting helps prevent such situations. <br><br>To enable this limit, set the property with a value bigger than zero. The default value is  `-1` (disabled). |
 | scripts.execution.maxMemoryUsedInBytes | The maximum memory (in bytes) a script is allowed to use. If script execution exceeds the configured memory, it will be stopped. <br><br>To enable this limit, set the property with a value bigger than zero. The default value is  `-1` (disabled). <br><br>If you would like to use this setting, 10000000 bytes (10 MB) is a reasonable value for custom scripts. This configuration only works with the supported JVM. |
-| scripts.execution.observerInstructionCount | The number of instructions that will trigger the observer that applies the memory and time limits. <br><br>The value may vary depending on the optimization level. <br><br>This configuration allows you to monitor the script execution and needs to be set to a value bigger than zero so that the described limits work. The default value is  `-1` (disabled). <br><br>This property is not linear, for example the instruction count here is not the number of Javascript instructions. A Javascript line can correspond to hundreds (or thousands) of lines for the observer. A value between 5000-10000 is suitable for this setting. |
+| scripts.execution.observerInstructionCount | The number of instructions that will trigger the observer that applies the memory and time limits. <br><br>The value may vary depending on the optimization level. <br><br>In Content Services 7.3.0, this configuration allows you to monitor the script execution and needs to be set to a value bigger than zero so that the described limits work. The default value is  `-1` (disabled). <br><br>Starting from Content Services 7.3.1, this configuration allows you to monitor the script execution and needs to be set to a value bigger than zero so that the time and memory limits work. The default value is `5000` so there is no need to initially change the property when enabling time or memory limits. <br><br>This property is not linear, for example the instruction count here is not the number of Javascript instructions. A Javascript line can correspond to hundreds (or thousands) of lines for the observer. A value between 5000-10000 is suitable for this setting. |
```

Comment:
```suggestion
| scripts.execution.observerInstructionCount | The number of instructions that will trigger the observer that applies the memory and time limits. <br><br>The value may vary depending on the optimization level. <br><br>In Content Services 7.3.0, this configuration allows you to monitor the script execution and needs to be set to a value bigger than zero so that the described limits work. The default value is  `-1` (disabled). <br><br>Starting from Content Services 7.3.1, this configuration allows you to monitor the script execution and must to be set to a value bigger than zero so that the time and memory limits work. The default value is `5000` so there is no need to initially change the property when enabling time or memory limits. <br><br>This property is not linear, for example the instruction count here is not the number of Javascript instructions. A Javascript line can correspond to hundreds (or thousands) of lines for the observer. A value between 5000-10000 is suitable for this setting. |
```